AcceptanceEKSJenkinsfile

```diff
@@ -132,8 +139,14 @@ pipeline {
                 '''
 
                 sh '''#!/bin/bash -ex
+                    echo 10.229.23.137 signin.aws.amazon.com >> /etc/hosts
+                    echo 10.229.23.151 signin.aws.amazon.com >> /etc/hosts
                     export ARTIFACTORY_READER=$FUNCTIONAL_USER_USR
                     export ARTIFACTORY_READER_PW=$FUNCTIONAL_USER_PSW
+                    pushd uaa-k8s-deploy
+                        chmod +x ./test/e2e-claim-test/set-env-qa.sh
```

Comment:
dont hardcode set-env-qa.sh, parameterize it with set-env-${DEPLOYMENT_ENVIRONMENT}.sh

Reply:
Done

Reply:
Done